vertical-pod-autoscaler/pkg/utils/vpa/capping_test.go

```diff
@@ -184,3 +184,57 @@ func TestApply(t *testing.T) {
 		assert.Equal(t, testCase.ExpectedError, err)
 	}
 }
+
+func TestApplyVpa(t *testing.T) {
```

Comment:
The behaviour of this test is difficult to understand.
Could you please add an initial comment about what's supposed to happen (CPU limited up, mem limited down) - it will help future readers.

Reply:
I have change the test a bit so that it should be more obvious. Let me know if you still want the comment.